docs/src/main/paradox/intellij-idea.md

```diff
@@ -0,0 +1,26 @@
+IntelliJ IDEA
```

Comment:
It's great to have that chapter in here (though we need to make sure it keeps current).

It also makes a good replacement for that old Akka 1.x page about how to use Scala IDE which until recently was still going strong in google... ;)